alf/environments/gym_wrappers_test.py

```diff
@@ -78,7 +78,7 @@ def test_framestack_all_fields(self):
             obs['dict']['inner_states'].shape,
         )
         expected = (
-            (2, 2, 3 * 4),  # 3 channels * 4
+            (3 * 4, 4, 4),  # 3 channels * 4
```

Comment:
what caused default image size to change from 2 to 4?

Reply:
because of the previous criterion for transpose:) As Wei commented above, we don't need that actually.